src/content/1.7/development/architecture/domain/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Domain
+menuTitle: Domain
+weight: 20
+---
+
+# Domain
```

Comment:
this has no sense to commit, then publish, then provide to Google and people empty pages, don't you think?

Reply:
Yeah, see my question about structuring above, I'm not sure where it should belong. When I get feedback, I'll update/change it. :)